lib/rules/require-comments-to-include-access.js

```diff
@@ -5,7 +5,7 @@ function isDocComment(comment) {
 }
 
 function includesAccessDeclaration(comment) {
-  return comment.value.match(/\n\s*(@private|@public)\s/);
+  return comment.value.match(/\n(\s*\*\s|\s*)(@private|@public)\s/);
```

Comment:
Actually on second thought, we shouldn't allow no whitespace (second example above is very bad).  Just multiples, which I think would be fixed by:

```javascript
return comment.value.match(/\n(\s*\*\s+|\s*)(@private|@public)\s/);
```

Along with another entry in the `good` test file for it.